### ISSUE TITLE
Add Datapoint name in the name of exported file

### ISSUE THREAD

**Issue by satblip** [infiniteautomation/ma-core-public#401]
Hi,

Would it be possible to add the Datapoint/Watchlist Name to the Excel Export.

That way we can avoid these kind of situation:

```
watchListData (1).xlsx
watchListData (2).xlsx
watchListData (3).xlsx
...
watchListData.xlsx
```

I suggest:

`Export-{datapoint/watchlist name}-{timestamp}.xls`

Thanks

**Comment by terrypacker** [comment]
I agree those are annoying, I'll get this into the 2.6.0 release.

**Comment by terrypacker** [comment]
Basic fix implemented in Core 2.6.0.  Since we are going to replace both pages in the near future the fix simply outputs a filename such as:

watchListData-2015-06-23_06-39-26.xlsx

or

pointData-2015-06-23_06-35-31.xlsx
